packages/react-components/theme-designer/src/components/Sidebar/Form.tsx

```diff
@@ -43,111 +60,215 @@ const useStyles = makeStyles({
-              onChange={handleKeyColorChange}
+      <Accordion defaultOpenItems={['1', '2', '3']} multiple>
+        {/* `multiple` allows for toggle of collapse as well as open multiple panels */}
+        <AccordionItem value="1">
+          <AccordionHeader>
+            <Caption1Stronger>Step 1 - Color settings</Caption1Stronger>
+          </AccordionHeader>
+
+          <AccordionPanel className={styles.accordionContainer}>
+            <div className={styles.inputs}>
+              <Label htmlFor={sidebarId + 'keyColor'}>Key color value</Label>
+              <div className={styles.labels}>
+                <Input
+                  className={styles.keyColor}
+                  size="large"
+                  appearance="underline"
+                  id={sidebarId + 'keyColor'}
+                  value={keyColor}
+                  onChange={handleKeyColorChange}
+                  maxLength={7}
+                  onBlur={handleKeyColorBlur}
+                />
+                <div className={styles.colorPicker} style={{ backgroundColor: keyColor }}>
+                  <input
+                    className={styles.color}
+                    type="color"
+                    id={sidebarId + 'keyColor Color'}
+                    onChange={handleKeyColorChange}
+                  />
+                </div>
+              </div>
+            </div>
+            <div>
+              <Label htmlFor={sidebarId + 'hueTorsion'}>Hue Torsion</Label>
+              <div className={styles.slider}>
+                <Slider
+                  size="small"
+                  min={-50}
+                  max={50}
+                  id={sidebarId + 'hueTorsion'}
+                  value={hueTorsion}
+                  onChange={handleHueTorsionChange}
+                />
+                <Input
+                  size="small"
+                  type="number"
+                  min={-50}
+                  max={50}
+                  appearance="outline"
+                  id={sidebarId + 'hueTorsion input'}
+                  value={hueTorsion.toString()}
+                  onChange={handleHueTorsionChange}
+                />
+              </div>
+            </div>
+            <div>
+              <Label htmlFor={sidebarId + 'vibrancy'}>Vibrancy</Label>
+              <div className={styles.slider}>
+                <Slider
+                  size="small"
+                  min={-50}
+                  max={50}
+                  id={sidebarId + 'vibrancy'}
+                  value={vibrancy}
+                  onChange={handleVibrancyChange}
+                />
+                <Input
+                  size="small"
+                  type="number"
+                  min={-50}
+                  max={50}
+                  appearance="outline"
+                  id={sidebarId + 'vibrancy input'}
+                  value={vibrancy.toString()}
+                  onChange={handleVibrancyChange}
+                />
```

Comment:
ditto Field usage, for the rest of the file

Reply:
Added Field, rem id prop